### PR TITLE
announcement of keycloak-dev-day in februrary 2024

### PR DESCRIPTION
**Pull request by srose**
Dear Keycloak-Team,

please find a proposal for an event announcement with this pull-request. We want to promote this community event to all enthusiastic Keycloak  users, contributors, maintainers, and so on. The event will happen in february 2024. 

As previously discussed, @ahus1 , please let me know if this is ok to publish.

All the best,
Sebastian

### CODE REVIEW

**Comment by ahus1** [comment]
Hi @srose - thank you for organizing this community event. I've added one commit to add more links, and update the headings from the questions to statements, as it would allow readers to scan the contents faster. 

Pull requests in the Keycloak organization now require a DCO signoff in their commits, see https://www.keycloak.org/2023/10/dco.html for the announcement. Please add it to your commit, and then this should be ready to go.

I'll wait for 1-2 more days to give others the chance for feedback.

**Review by ahus1 — APPROVED** [review]
Thank you @srose for adding the DCO signoff - added my review commit again, and it should now be ready to be merged.

**Review by stianst — APPROVED** [review]

**Pull request merged**
